packages/web/src/types.ts

```diff
@@ -143,3 +143,5 @@ export interface analyticsConfig {
 	userId: string;
 	customEvents: object;
 }
+
+export type showClearAll = 'never' | 'always' | 'default' | true | false;
```

Comment:
@ShahAnuj2610 Can you use constants here?

Reply:
it's a TS file, not able to import it